src/index.js

```diff
@@ -205,6 +209,14 @@ async function notifySprintsBacklogs() {
   return dataToSend;
 }
 
+/**
+ * Meeting Message to display
+ */
+const MEETING_MSG = `☝️ Join the meeting in Discord!
+@specc @guryn @khaydarovm @nikmel2803 @xemk4
```

Comment:
it would be great if this list will be created automatically by MENTION list may be